lib/assessment.js

```diff
@@ -5,15 +5,19 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  for (i=0; i< arguments.length; i++){
+    return arguments+
+  }
+min(1,4,9);
 
-};
+// I know this is wrong, I need to put args in min(args) but cant remember how to set it to the next variable because when I type in arg = arguments, arugments is already defined and my code will only accept three cases right now because of how its set up as a simple if/then statement.
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
-
+var sum = function sum(arg1,arg2) {
+  return arg1 + arg2;
```

Comment:
Now imagine that the concept from Q1 is applied to Q2. You might get something like this:
```javascript
var sum = function(){
  var total = 0;
  for (var i = 0; i < arguments.length; i++) {
    total += arguments[i];
  }
  return total;
};
```